data/build.gradle

```diff
@@ -43,9 +39,6 @@ android {
         test {
             java.srcDirs += 'src/test/kotlin'
         }
-        staging_internal {
-            java.srcDirs += 'src/staging'
-        }
```

Comment:
I'm actually not very familiar what's going on here, but instead of removing it completely, should it be like this instead?

```
staging {	
    java.srcDirs += 'src/staging'	
}
```

🙏

Reply:
The Android Framework knows where to look for a default source set. In this case, we did not want to use the default source set location but instead add the staging folder to be shared with the staging_internal flavor.

So to answer your question, no it should not be like this instead.

You can read more about how the framework behaves for [default](https://developer.android.com/studio/build/build-variants#configure-sourcesets) source set folders, and we can discuss further if you feel the need.

Reply:
Aha, got it! Thanks